ui/app/src/App.tsx

```diff
@@ -3,14 +3,21 @@ import { useState } from "react";
 import { Stream } from "@apeworx/apepay";
 import { TokenInfo } from "@uniswap/token-lists";
 import { ConnectButton } from "@rainbow-me/rainbowkit";
-
-// import { CreateStream, StreamStatus } from "@apeworx/apepay-react";
-import { StreamStatus } from "@apeworx/apepay-react";
-import CreateStream from "../../../ui/lib/CreateStream";
+import {
+  usePublicClient,
+  useWalletClient,
+  WalletClient,
+  useAccount,
+} from "wagmi";
 import config from "./config";
 // NOTE: Do this or else it won't render (or create your own CSS)
 import "rc-slider/assets/index.css";
 import "./styles.css";
+// import { CreateStream, StreamStatus } from "@apeworx/apepay-react";
+// import { StreamStatus } from "@apeworx/apepay-react";
+import CreateStream from "../../../ui/lib/CreateStream";
+import StreamStatus from "../../../ui/lib/StreamStatus";
+import StreamStatusBar from "../../../ui/lib/StreamStatusBar";
 
```

Comment:
Would really like to fix this so that its not local imports

Reply:
Tried a bunch of different things; @mikeshultz says this is for monorepos; with his help I aliased it.